src/renderer/VideoButton.tsx

```diff
@@ -168,32 +169,34 @@ export default function VideoButton(props: any) {
 
   const buttonClasses = ['videoButton'];
   let keystoneTimelineSegments = [];
-
+  
   if (isMythicPlus) {
+    console.log("ahk");
```

Comment:
ahk? ahk!

Reply:
Printing your initials is better than "hello" right?